src/main/java/org/openstreetmap/josm/plugins/pt_assistant/data/PTStop.java

```diff
@@ -433,7 +433,7 @@ public static boolean isPTStopPosition(RelationMember rm) {
     public static boolean isPTPlatform(RelationMember rm) {
         return rm.getMember().hasTag("highway", "bus_stop") || rm.getMember().hasTag("public_transport", "platform")
                 || rm.getMember().hasTag("highway", "platform") || rm.getMember().hasTag("railway", "platform")
-                || rm.getMember().hasTag("amenity", "bus_station");
+                || rm.getMember().hasTag("amenity", "bus_station") || rm.getMember().hasTag("highway", "tram_stop");
```

Comment:
`highway=tram_stop` is not really a thing as far as I am aware?

Tram stop positions are tagged as [railway=tram_stop](https://wiki.openstreetmap.org/wiki/Tag:railway%3Dtram_stop) / [public_transport=stop_position](https://wiki.openstreetmap.org/wiki/Tag:public_transport%3Dstop_position) and platforms as [highway=platform](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dplatform) / [railway=platform](https://wiki.openstreetmap.org/wiki/Tag:railway%3Dplatform) / [public_transport=platform](https://wiki.openstreetmap.org/wiki/Tag:public_transport%3Dplatform)

It only has one occurrence on taginfo
https://taginfo.openstreetmap.org/search?q=highway%3Dtram_stop

Reply:
`railway=tram_stop` is also not a valid tag for a platform and should not be considered valid here

https://wiki.openstreetmap.org/wiki/Tag:railway%3Dtram_stop
https://josm.openstreetmap.de/ticket/17378#comment:2

Reply:
Sorry, I made the pull request late last night. Will correct.